### ISSUE TITLE
Incorrect DataTable format in HTML report

### ISSUE THREAD

**Issue by shell-software** [etorreborre/specs2#466]
Hi,

It seems that there was a similar [issue #106](https://github.com/etorreborre/specs2/issues/106).
However I'm still having troubles.

This is what I have in my **Spec** class:

```scala
import org.specs2.Specification
import org.specs2.specification.core.SpecStructure
import org.specs2.matcher.DataTables
class SampleSpec extends Specification with DataTables {

	override def is: SpecStructure =
		"Testing web-socket" ! checklist

	def checklist =
		"URI"                       ||   "Path"     |   "Message"     |   "Expected"     |>
		"ws://echo.websocket.org"   !!   ""         !   "hello"       !   "hello"        |
		"ws://echo.websocket.org"   !!   ""         !   "hello"       !   "hellofailed"  |
		"ws://echo.websocket.org"   !!   ""         !   "buy"         !   "buy"          | {
			(uri, path, message, expected) => {
				getResponse(uri, path, message) === actual
			}
	}

}
```
Execution produces the following report:

![report](https://cloud.githubusercontent.com/assets/10495282/14074977/d9513f86-f4dd-11e5-99fd-1318817ee335.png)

As you see **DataTable** is shown like a simple string without any formatting

Do I missing something or is it an issue?

**Comment by shell-software** [comment]
Many thanks

**Comment by etorreborre** [comment]
You didn't miss anything, it was an oversight on my side when I did the refactoring of the Html printer last year. This is now fixed in `3.7.2-20160324133853-454b358`. Please tell me if you find some remaining issues because formatting is a tricky business.

**Comment by shell-software** [comment]
Sure
Thanks for quick response
